### ISSUE TITLE
Duplicate timestamp displaying on MAP pages where lastPublished and firstPublished are the same day

### ISSUE THREAD

**Issue by karinathomasbbc** [bbc/simorgh#5509]
This was discovered to be a pan-simorgh issue. Discussion thread: https://bbcnews.slack.com/archives/CK85Q5SL9/p1581500636104900

Tl;dr **If the firstPublished date = the lastPublished date and it’s not today, then only show the firstPublished date**

---

Pre-requisite: Legacy MAPs on Simorgh on Test environment

1. Navigate to https://www.test.bbc.com/arabic/multimedia/2014/05/140515_kenia_children_sex_trade
(https://ares-broker.api.bbci.co.uk/api/asset/arabic/multimedia/2014/05/140515_kenia_children_sex_trade for data)
2. Right click and "Translate into English"
3. Note that there are 2 timestamps below the headline.
![image](https://user-images.githubusercontent.com/58214768/74262284-7f517f00-4cf4-11ea-8e42-c6761a1d870d.png)

Expected: Since the timestamp & updated is the same date, should only 1 be displayed?
On PAL, only a single timestamp is displayed - https://pal.live.bbc.com/arabic/multimedia/2014/05/140515_kenia_children_sex_trade
